payments-core/src/main/java/com/stripe/android/googlepaysheet/GooglePaySheetLauncher.kt

```diff
@@ -15,7 +15,7 @@ import kotlin.coroutines.CoroutineContext
 
 internal interface GooglePaySheetLauncher {
```

Comment:
Will the other GooglePaySheet* be renamed too?

Reply:
I'll do that in a future PR.